RELEASENOTES.md

```diff
@@ -1 +1,8 @@
+- Improve error messages when the specified target organization or enterprise cannot be found
+- Mask the value for `AWS_ACCESS_KEY` and `AWS_SECRET_KEY` parameters in log output
+- Fix log output so we don't say we've finished upload to Azure Blob Storage when you're actually using Amazon S3
+- Extend the expiration of blob storage signed URLs from 24hrs to 48hrs so migration can still be successful even if there is a long queue of migrations
+- Skip the upload to Azure/AWS blob storage when migrating from GHES 3.8+, as GHES will now handle putting the archives into blob storage
+- Fixed a bug where bad credentials were incorrectly being treated as rate-limit errors
```

Comment:
Should this all be here? I guess this happened because of a merge conflict.

Reply:
Addressed in https://github.com/github/gh-gei/pull/787/commits/ed65a969f41f9ab96f6811fc9b8ff9410000f87a :tada: